Tools/clinic/clinic.py

```diff
@@ -2235,11 +2246,7 @@ def parse(self, input: str) -> str:
                     assert dsl_name in parsers, f"No parser to handle {dsl_name!r} block."
                     self.parsers[dsl_name] = parsers[dsl_name](self)
                 parser = self.parsers[dsl_name]
-                try:
-                    parser.parse(block)
-                except Exception:
-                    fail('Exception raised during parsing:\n' +
-                         traceback.format_exc().rstrip())
```

Comment:
pyflakes now tells me:

```
Tools/clinic\clinic.py:31:1: 'traceback' imported but unused
```

Reply:
Huh. This code is removed by this PR.

Reply:
Yes, exactly: you have removed the only use of the `traceback` module from `clinic.py`, meaning that the import statement at the top of the file is now unused :)

Reply:
Ahh 😆 I thought you meant an unused variable or something like that; sorry 'bout that!

Reply:
Removed!